cilium-health/launch/endpoint.go

```diff
@@ -52,16 +52,6 @@ func logFromCommand(cmd *exec.Cmd, netns string) error {
 			scopedLog.Debugf("%s", in.Text())
 		}
 	}()
-	stderr, err := cmd.StderrPipe()
```

Comment:
I think the right answer here is probably to restrict the permissions set to only one of the daemons.

In the log you can see that one of the logs states "netns=cilium-health", while the other doesn't. Both the cilium-health instance that runs on the host, and the cilium-health that runs in a namespace are setting the permissions of this path, but they don't both need to - only one of them is actually hosting that socket. The permissions set should be contingent upon serving the unix api.

Reply:
fixed by dropping this commit